### ISSUE TITLE
Add holidays

### ISSUE THREAD

**Issue by leobetosouza** [jmeas/moment-business#46]
I know that holidays varies by country, state, city, province, etc

I think would be good have a way to set the holidays, with a array of month/date

For instance with the [US federal holidays](https://www.redcort.com/us-federal-bank-holidays/) would be something like this `moment.setHolidays(['January 01', 'January 18', 'February 15', 'July 04', 'May 30', 'September 5', 'October 10', 'November 11', 'November 26', 'December 26'])`

For [NY](http://publicholidays.us/new-york/) we would push some more days to the list: `moment.addHolidays(['February 12', 'November 08', 'December 25'])`

Them when we use methods of `moment-bussiness` it consider these days not as weekdays, and not as weekendDays. 

We can also create methods to count holidays.

What do you think?

**Comment by jamesplease** [comment]
Makes sense to me. If you've got time to open a PR, I would review/merge it :v: But I won't have time to add the feature myself

**Comment by jakeleboeuf** [comment]
@jmeas anybody ever take a stab at a PR for this?

**Comment by jamesplease** [comment]
Not yet 🙂

**Comment by gndelia** [comment]
Is this alive ? If I created a PR, would there any time to review it?
